CHANGELOG.md

```diff
@@ -24,6 +24,12 @@ Please visit [cucumber/CONTRIBUTING.md](https://github.com/cucumber/cucumber/blo
 
 ### Security fixes
 
+## [NEW VERSION](https://github.com/cucumber/cucumber-ruby/compare/v5.2.0...v5.3.0)
```

Comment:
Just add this under in git. Then once we cut we just alter the version candidate accordingly.